optimum/habana/transformers/models/bert/modeling_bert.py

```diff
@@ -119,3 +126,78 @@ def gaudi_BertModel_forward(
         attentions=encoder_outputs.attentions,
         cross_attentions=encoder_outputs.cross_attentions,
     )
+
+
+def gaudi_Bert_Sdpa_SelfAttention_forward(
+    self,
+    hidden_states: torch.Tensor,
+    attention_mask: Optional[torch.Tensor] = None,
+    head_mask: Optional[torch.FloatTensor] = None,
+    encoder_hidden_states: Optional[torch.FloatTensor] = None,
+    encoder_attention_mask: Optional[torch.FloatTensor] = None,
+    past_key_value: Optional[Tuple[Tuple[torch.FloatTensor]]] = None,
+    output_attentions: Optional[bool] = False,
+) -> Tuple[torch.Tensor]:
+    r"""
+    Copied from https://github.com/huggingface/transformers/blob/v4.46.3/src/transformers/models/bert/modeling_bert.py
```

Comment:
Has it been updated with Transformers v4.51.3? Since the codebase is now aligned with that version of Transformers